app/views/api_docs/vendor_api_docs/reference/draft.html.erb

```diff
@@ -0,0 +1,170 @@
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent"><%= govuk_link_to 'Version 1.1 changes', '#important', class: 'app-contents-list__link' %></li>
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent"><%= govuk_link_to 'Developing on the API', '#developing', class: 'app-contents-list__link' %></li>
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent">
+    <%= govuk_link_to 'Endpoints', '#endpoints', class: 'app-contents-list__link' %>
+
+    <ol class="app-contents-list__nested-list">
+    <% @api_reference.operations.each do |operation| %>
+      <li class="app-contents-list__list-item app-contents-list__list-item--dashed">
+        <%= govuk_link_to operation.name, "##{operation.anchor}", class: 'app-contents-list__link' %>
+      </li>
+    <% end %>
+    </ol>
+  </li>
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent">
+    <%= govuk_link_to 'Objects', '#objects', class: 'app-contents-list__link' %>
+
+    <ol class="app-contents-list__nested-list">
+      <% @api_reference.schemas.each do |schema| %>
+        <li class="app-contents-list__list-item app-contents-list__list-item--dashed">
+          <%= govuk_link_to schema.name, "##{schema.anchor}", class: 'app-contents-list__link' %>
+        </li>
+      <% end %>
+    </ol>
+  </li>
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent">
+    <%= govuk_link_to 'Field lengths summary', '#field-lengths', class: 'app-contents-list__link' %>
+  </li>
+</ol>
+
+<hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+
+<h2 class="govuk-heading-l" id="versions">API Versions</h2>
+
+<p class="govuk-body">
+  The current version of this API is 1.0. The next version of this API will be 1.1.  This is a draft specification for version 1.1.
+</p>
+
+<p class="govuk-body">
+  From version 1.1 onwards, version numbers will follow this format: MAJOR.MINOR (e.g. 1.2)
+</p>
+
+<p class="govuk-body">
+  MAJOR version: The version used in the URI and defines breaking changes to the API.
+</p>
+
+<p class="govuk-body">
+  MINOR version: Includes non-breaking changes to the API, mostly additions. For example, a minor version may add a new field or a new endpoint.
+</p>
+
+<p class="govuk-body">
+  All changes, minor or major, will be documented in our release notes.
+</p>
+
+<p class="govuk-body">
+Minor version updates will become available automatically at the MAJOR version urls. For example, version 1.1 urls will be available automatically under /api/v1 when version 1.1 is released. Pre-existing urls may return additional fields at this point.
+
+<p class="govuk-body">
+We will also support requesting a specific minor version via the url, see examples of this within the <%= govuk_link_to 'Developing on the API', '#developing', class: 'app-contents-list__link' %> section.
```

Comment:
discussed in person: we should explain the URI versioning here instead of the developing the API section